localization/src/main/kotlin/edu/artic/localization/LocalizationPreferences.kt

```diff
@@ -14,6 +14,10 @@ import java.util.*
 class LocalizationPreferences(context: Context)
     : BasePreferencesManager(context, "localization") {
 
+    companion object {
+        const val TOUR_KEY = "tour_locale"
```

Comment:
The name looks an awful lot like `ARG_TOUR`. Do you think the fact that this is in `LocalizationPreferences.kt` will be enough to distinguish the two?

Reply:
That name is awful 😄.  Updated it to PREF_TOUR_LOCALE. You are absolutely right sir.